src/grandorgue/gui/GOGUISizeKeeper.cpp

```diff
@@ -21,6 +21,15 @@ const wxString WX_WINDOW_WIDTH = wxT("WindowWidth");
 const wxString WX_WINDOW_HEIGHT = wxT("WindowHeight");
 const wxString WX_DISPLAY_NUMBER = wxT("DisplayNumber");
 const wxString WX_WINDOW_MAXIMIZED = wxT("WindowMaximized");
+const wxString WX_N_ADD_SIZES = wxT("NAddSizes");
```

Comment:
Most other cmb settings have "NumberOf" as prefix and one has Count suffix (SavedCount= used for dialog sizes). I can't help but feel that the naming used here is strange and unnecessarily shortened. Maybe to be consistent with the other dialog sizes "AdditionalSizeCount" would be better.